### ISSUE TITLE
proj_use_flow()

### ISSUE THREAD

**Issue by ijlyttle** [ijlyttle/projthis#18]
create root-level directories:

```
dataflow
workflow
```

Add these directories to `.rbuildignore`, optionally add dataflow to `.gitignore`

**Comment by ijlyttle** [comment]
```r
proj_use_flow <- function(gitignore_data = TRUE) {

  usethis::use_directory(usethis::proj_path("workflow"))
  usethis::use_directory(usethis::proj_path("dataflow"))

  if (gitignore_data) {
    usethis::use_gitignore(usethis::proj_path("dataflow"))
  }

  invisible(NULL)
}
```

**Comment by ijlyttle** [comment]
closed in service of larger work merged in #20
